src/components/login/index.js

```diff
@@ -0,0 +1,50 @@
+import React, { PropTypes, Component } from 'react'
+import { translate } from 'react-i18next'
+
+class Login extends Component {
+	constructor (props) {
+		super(props)
+		this._handleSubmit = this.handleSubmit.bind(this)
```

Comment:
@Fiaxhs thats what I do now. But I'm not sure it's handled natively yet, you may need babel for this (it's es6 stage2 https://babeljs.io/docs/plugins/transform-class-properties/)

@Shahor I'm just telling you yo bind yourself, but with the same actual method name. binded handleSubmit replace handleSubmit, and you don't need to have another method prefixed with _ for that